spk/lidarr/Makefile

```
@@ -5,12 +5,19 @@ SPK_ICON = src/lidarr.png
 
 DEPENDS = cross/lidarr cross/libstdc++ cross/sqlite
 
-UNSUPPORTED_ARCHS = $(DOTNET_UNSUPPORTED_ARCHS)
+UNSUPPORTED_ARCHS = $(PPC_ARCHS) $(ARMv5_ARCHS) $(ARMv7L_ARCHS)
+# custom exclusions for dotNET 6 on ARMv7 archs
+include ../../mk/spksrc.common.mk
+ifeq ($(call version_lt, ${TCVERSION}, 7.0),1)
+UNSUPPORTED_ARCHS += $(ARMv7_ARCHS)
+else
+UNSUPPORTED_ARCHS += armada370
+endif
 
 MAINTAINER = Team Lidarr
 MAINTAINER_URL = https://lidarr.audio/
 DESCRIPTION  = Lidarr is a music collection manager for Usenet and BitTorrent users. It can monitor multiple RSS feeds for new tracks from your favorite artists and will grab, sort and rename them. It can also be configured to automatically upgrade the quality of files already downloaded when a better quality format becomes available.
-CHANGELOG = "1. Update Lidarr to v1.0.2.<br/>2. Use alternate TMP directory to provide enough space for internal updater.<br/>3. Enlarge the service start/stop timeout to 90 seconds.<br/>4. Avoid Lidarr downgrade on package update."
+CHANGELOG = "1. Fix package update.<br>2. Add support for x86 (32-bit) architecture."
```

```suggestion
CHANGELOG = "1. Fix package update.<br/>2. Add support for x86 (32-bit) architecture."
```

You need the `/` otherwise bad stuff happens, unfortunately I don't remember more about it.

---

Thanks for the feedback. If you can, I'd really like your help validating the need for this because I have two bits of information which conflict with this suggestion:

1. The example for the `CHANGELOG` in our own Wiki page on [Available Variables in spk/ Makefiles](https://github.com/SynoCommunity/spksrc/wiki/Makefile-variables#available-variables-in-spk-makefiles) shows `<br>` separating multiple entries
2. The W3C Markup Validation Service has some documentation on [Trailing slashes](https://github.com/validator/validator/wiki/Markup-%C2%BB-Void-elements#trailing-slashes) which indicates that "_a trailing slash to mark start tags as self-closing_"

As such, I'd really like some clarity on the correct use of HTML with void elements in the change log.

---

so I got a response from @hgy59 on this (https://github.com/SynoCommunity/spksrc/pull/5590#discussion_r1095387109) and I'll ensure that all my code has this changed.